makeybutton/button.go

```diff
@@ -47,6 +44,15 @@ func NewButton(pin machine.Pin) *Button {
 	}
 }
 
+// Configure configures the Makey Button pin to have the correct settings to detect touches.
+func (b *Button) Configure() error {
+	b.pin.Configure(machine.PinConfig{Mode: machine.PinInputPullup})
```

Comment:
Is this needed if the next line changes the mode too ?

Reply:
It is needed to set the pin into the correct floating mode. A little weird, but needed.

Reply:
If a comment or other information is not added, it will be accidentally erased during future changes.
So, I would suggest adding a comment.

Reply:
Added.